### PR TITLE
Adding support for multiple auth credentials for nginx (OPS-952)

### PR DESCRIPTION
**Pull request by bilalahmad99**

### CODE REVIEW

File: playbooks/roles/nginx/defaults/main.yml [hunk 1]
```diff
@@ -10,6 +10,15 @@ NGINX_EDXAPP_EXTRA_SITES: []
 NGINX_EDXAPP_EXTRA_CONFIGS: []
 NGINX_EDXAPP_CUSTOM_REDIRECTS: {}
 
+# Override these vars for adding user to nginx.htpasswd
+nginxusers:
+  - name: "{{ COMMON_HTPASSWD_USER }}"
+    password: "{{ COMMON_HTPASSWD_PASS }}"
+    state: present
+  - name: bilal
```
Comment (feanil) [inline playbooks/roles/nginx/defaults/main.yml]: You should remove your self from the default settings.  Having a list of one as the default should be sufficient.

Comment (bilalahmad99) [inline playbooks/roles/nginx/defaults/main.yml]: Done

File: playbooks/roles/nginx/defaults/main.yml [hunk 2]
```diff
@@ -10,6 +10,12 @@ NGINX_EDXAPP_EXTRA_SITES: []
 NGINX_EDXAPP_EXTRA_CONFIGS: []
 NGINX_EDXAPP_CUSTOM_REDIRECTS: {}
 
+# Override these vars for adding user to nginx.htpasswd
+nginxusers:
```
Comment (e0d) [inline playbooks/roles/nginx/defaults/main.yml]: NGINXUSERS should be uppercase as we expect it to be overridden

Comment (bilalahmad99) [inline playbooks/roles/nginx/defaults/main.yml]: Done

**Comment by e0d** [comment]
:+1:

**Pull request merged**